File: app/src/main/java/ir/touristland/Models/HotelRoomPossibilitiesItem.java
package ir.touristland.Models;

import java.io.Serializable;

public class HotelRoomPossibilitiesItem implements Serializable {

    public static final long serialVersionUID = 1L;
    public String Id = "";
    public String HotelRoomId = "";
    public String CenterId = "";
    public String RoomPossibilitiesId = "";
    public String RoomPossibilitiesName = "";
    public String IconName = "";

    public String getId() {
        return Id;
    }

    public String getHotelRoomId() {
        return HotelRoomId;
    }

    public String getCenterId() {
        return CenterId;
    }

    public String getRoomPossibilitiesId() {
        return RoomPossibilitiesId;
    }

    public String getRoomPossibilitiesName() {
        return RoomPossibilitiesName;
    }

    public String getIconName() {
        return IconName;
    }

}

File: app/src/main/java/ir/touristland/Models/HotelRoomPossibilitiesItem.kt
package ir.touristland.Models

import java.io.Serializable

class HotelRoomPossibilitiesItem : Serializable {
    var id = ""
    var hotelRoomId = ""
    var centerId = ""
    var roomPossibilitiesId = ""
    var roomPossibilitiesName = ""
    var iconName = ""

    companion object {

        const val serialVersionUID = 1L
    }

}
